raft/raft_test.go

```diff
@@ -2914,6 +2914,44 @@ func TestLearnerReceiveSnapshot(t *testing.T) {
 	}
 }
 
+// TestNormalPeerRejectLearnerSnapshot tests that normal peer should
```

Comment:
shouldnt we test `If learner just has been added by conf change, it should accept leader snapshot.` instead?